src/ripple_txt.js

```diff
@@ -7,7 +7,7 @@ export default class RippleTxt {
 
     return new Promise((resolve, reject) => {
       RippleVaultClient.RippleTxt.get(domain, (error, resp) => {
-        if (Object.keys(resp).length === 0 || !resp) {
+        if (!resp || Object.keys(resp).length === 0) {
```

Comment:
The nodejs convention is to always define either `error` or `resp` but not both.